api/tests/integration/infrastructure/repositories/assessment-repository_test.js

```diff
@@ -450,80 +416,41 @@ describe('Integration | Infrastructure | Repositories | assessment-repository',
       return databaseBuilder.commit();
     });
 
-    context('When several assessments exist for a given certification course id', () => {
+    context('When an assessment exists for this certificationCourseId', () => {
```

Comment:
Je vois pas la différence entre le contexte de la ligne 340 et 419 ? Il ne devrait pas y avoir un cas négatif plutôt ? 
Genre : when there is no assessment for a certificationCourseId ?

Reply:
Oui c'est deux fois les mêmes contextes, mais dans le test de deux méthodes différentes ? Du coup je comprends pas la question oO